ivy_tests/test_ivy/test_frontends/test_numpy/test_statistics/test_averages_and_variances.py

```diff
@@ -302,3 +302,55 @@ def test_numpy_cov(
         dtype=dtype[0],
         test_values=False,
     )
+
+
+# nanmedian
+@handle_frontend_test(
+    fn_tree="numpy.nanmedian",
+    dtype_and_a=statistical_dtype_values(function="nanmedian"),
```

Comment:
make `function=mean` and see if it works

Reply:
Okay